src/styles/card/card.scss

```diff
@@ -0,0 +1,113 @@
+.main {
+  padding: 50px 40px;
+  height: 862px;
+  max-width: 1200px;
+
+  /* Центрування елемента горизонтально */
+  margin: 0 auto;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  grid-template-rows: repeat(2, 408px);
+  gap: 46px 48px;
+
+  justify-content: center;
+  align-items: center;
+}
+
+@media (min-width: 488px) {
+  .main {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .main {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .main {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
```

Comment:
It's a good practice to avoid hardcoding values. For the grid-template-columns property, you could use '1fr' instead of '200px' to make your grid more flexible and responsive.